cifti/afni_xml_io.h

```diff
@@ -50,26 +50,50 @@
     - convert as in SUMA_Create_Fake_CIFTI()
  * ----------------------------------------------------------------------*/
 
+#ifndef CIF_API
+   #if defined(_WIN32) || defined(__CYGWIN__)
+      #if defined(CIFTI_BUILD_SHARED)
+      #ifdef __GNUC__
+         #define CIF_API __attribute__ ((dllexport))
+      #else
+         #define CIF_API __declspec( dllexport )
+      #endif
+      #elif defined(CIFTI_USE_SHARED)
```

Comment:
I'm not sure these BUILD_SHARED / USE_SHARED conditions are needed... at least, other projects like [ITK](https://github.com/InsightSoftwareConsortium/ITK/blob/3ada77dd641c14e1dfd5a9a6f18f738eea148838/Modules/Core/Common/include/itkMacro.h#L210) and [VTK](https://github.com/Kitware/VTK/blob/e19328f6fce9a51f992fff5dbde5a3b8a82209ff/Common/Core/vtkABI.h#L37) don't seem to have them.

Seems would be simpler without them.

Reply:
Huh, it does seem to work using only dllexport

Reply:
I'm not sure how this works exactly, e.g. [zlib](https://github.com/madler/zlib/blob/0f51fb4933fc9ce18199cb2554dacea8033e7fd3/zconf.h#L334) also seems to do it like I do (kinda, I think they define the ZLIB_INTERNAL inside their .c files when including the zlib.h header)

Reply:
Basically, it's harmless to attach the 'export' attribute in the case of building for static libraries, so simpler to always do it.